plugins/GSdx/GSState.cpp

```diff
@@ -433,8 +433,16 @@ GSVector4i GSState::GetDisplayRect(int i)
 	rectangle.right = rectangle.left + width;
 	rectangle.bottom = rectangle.top + height;
 
-	// Useful for debugging games:
-	//printf("DW: %d , DH: %d , left: %d , right: %d , top: %d , down: %d , MAGH: %d , MAGV: %d\n", m_regs->DISP[i].DISPLAY.DW, m_regs->DISP[i].DISPLAY.DH, r.left, r.right, r.top, r.bottom , m_regs->DISP[i].DISPLAY.MAGH,m_regs->DISP[i].DISPLAY.MAGV);
```

Comment:
Uncommenting it only isn't enough to make it work anyway. I tried to use it ages ago for a quick test, but then the `r.` had already been replaced by `rectangle` and it has down instead of bottom which was a little confusing. :p